### PR TITLE
Fix #37043 celtabettv5.com

### LINKED ISSUE #37043: celtabettv5.com
### Issue URL (Ads)
[https://www.celtabettv5.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.celtabettv5.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.1.2817.0
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Block third-party Authorization header, Self-destructing third-party cookies (0), Disable cache for third-party requests, Block WebRTC, Block Push API, Block Location API, Block Flash, Block Java
Filters: | AdGuard Russian,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/37043

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -83,6 +83,9 @@ tekkocaeli.com##.sagkulereklam
 ||tekkocaeli.com/wp-content/uploads/2018/10/377_8.jpg
 ||gercekkocaeli.com.tr/resimler/images/Teknocom(1).jpg
 gercekkocaeli.com.tr##ul[style="width:400px; height:300px;"]
+celtabettv5.com#$#body { background-image: none!important; }
+celtabettv5.com##.siteAnnouncer
+||celtabettv5.com/*/banners/*.gif
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt]: use existing `/uploads/banners/*$domain=`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (tvinzz) [reply TurkishFilter/sections/specific.txt]: added domain to existing rule

Comment (tvinzz) [reply TurkishFilter/sections/specific.txt]: moved #$#-rule to general_extensions

**Review by Alex-302 — APPROVED** [review]

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -83,6 +83,9 @@ tekkocaeli.com##.sagkulereklam
 ||tekkocaeli.com/wp-content/uploads/2018/10/377_8.jpg
 ||gercekkocaeli.com.tr/resimler/images/Teknocom(1).jpg
 gercekkocaeli.com.tr##ul[style="width:400px; height:300px;"]
+celtabettv5.com#$#body { background-image: none!important; }
```
[the remark this answers is not in the document]
Comment (tvinzz) [reply TurkishFilter/sections/specific.txt]: removed redundant rule

**Review by AdamWr — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Pull request merged**